verification/curator-service/ui/src/components/LinelistTable.test.tsx

```diff
@@ -59,6 +59,57 @@ it('loads and displays cases', async () => {
   expect(items).toBeInTheDocument();
 })
 
+it('API errors are displayed', async () => {
+  const cases = [
+    {
+      _id: 'abc123',
+      importedCase: {
+        outcome: 'Recovered',
+      },
+      location: {
+        country: "France",
+      },
+      events: [
+        {
+          name: 'confirmed',
+          dateRange: {
+            start: new Date().toJSON(),
+          },
+        }
+      ],
+      notes: "some notes",
+      source: {
+        url: "http://foo.bar",
+      }
+    },
+  ];
+  const axiosResponse = {
+    data: {
+      cases: cases,
+      total: 15,
+    },
+    status: 200,
+    statusText: 'OK',
+    config: {},
+    headers: {},
+  };
+  mockedAxios.get.mockResolvedValueOnce(axiosResponse);
+
+  const { getByText, findByText } = render(<LinelistTable />)
+
+  // Throw error on add request.
+  mockedAxios.post.mockRejectedValueOnce(new Error("Request failed"));
```

Comment:
nit but wouldn't the test be simpler and test the same thing if the initial get response was returning the error?

Reply:
Yes that was my initial thought, but the table rendering is not failing gracefully when the get errors out. We should fix this, but in a different PR IMO.

Reply:
ah ok good to know (and to fix), lgtm